src/main/webapp/app/views/inputtype/input-contact.html

```diff
@@ -11,7 +11,7 @@
     typeahead-no-results="noResults"
     typeahead-on-select="saveWithCV(fieldValue, $item)"/>
   <span class="input-group-btn">
-    <button class="btn btn-danger submission-input-contact-remove" ng-if="fieldValue.value">
+    <button class="btn btn-default opaque" ng-if="fieldValue.value">
```

Comment:
You changed the `ng-if` to `ng-show` elsehwere.
Why is this one not done as well?